<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5UH8T1U" ignored="0" name="Test" tests="3" failures="1" timestamp="2021-10-17T12:17:23 IST" time="90.629" errors="0">
  <testcase name="picsartUIValidation" time="24.484" classname="interview.PicsartTest">
    <skipped/>
  </testcase> <!-- picsartUIValidation -->
  <testcase name="picsartUIValidation" time="25.285" classname="interview.PicsartTest">
    <skipped/>
  </testcase> <!-- picsartUIValidation -->
  <testcase name="picsartUIValidation" time="30.921" classname="interview.PicsartTest">
    <failure type="java.lang.AssertionError" message="expected [https://picsart.com/blog/category/design-school] but found [https://picsart.com/blog]">
      <![CDATA[java.lang.AssertionError: expected [https://picsart.com/blog/category/design-school] but found [https://picsart.com/blog]
at pageClass.BlogPage.validateBlogPage(BlogPage.java:95)
at interview.PicsartTest.picsartUIValidation(PicsartTest.java:60)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- picsartUIValidation -->
</testsuite> <!-- Test -->
